packages/mui-material/src/Paper/Paper.d.ts

```diff
@@ -7,37 +7,39 @@ import { PaperClasses } from './paperClasses';
 
 export interface PaperPropsVariantOverrides {}
 
+export interface PaperBaseProps {
```

Comment:
Is `…Base…Props` used in other places? I have these props in Joy UI as well but with `…Static…Props`. If we are agreed to use `…Base…Props`, I will change Joy types. cc @mui/core

Reply:
Yeah, Base UI uses ...OwnProps pretty consistently.

Reply:
Could you please update it to `PaperOwnProps`? Typography and FormLabel use this pattern as well.
I wouldn't use `BaseProps` as we've got some components with the `Base` suffix in Material UI and this could create some confusion (or lead to abominations like `ButtonBaseBaseProps` in the future).

Reply:
done in these commits https://github.com/mui/material-ui/pull/35924/commits/d106afbbba0b6328e25168d42b4c0ff52c7f1f67 https://github.com/mui/material-ui/pull/35924/commits/9a6533bb32bb79a6e43109a1984bdee6505e6677